components/tekton/tasks/deploy-gotests.yaml

```diff
@@ -32,17 +33,21 @@ spec:
       type: string
     - name: FEATURE
       type: string
+    - name: SMOKE
```

Comment:
Maybe you can change the name a bit or add a description to explain what this param is for?

Reply:
added  [here](https://github.com/jnunyez/vse-catalog/commit/5338e95a076e7bdaf0e0f99e47b574c497726d9b)